test/evergreen.yml

```diff
@@ -315,6 +315,18 @@ functions:
       permissions: public-read
       content_type: text/plain
       remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${task_name}_${build_id}/
+  "upload stacktrace subtest": &upload_stacktrace_subtest
```

Comment:
@tod-johnson-mongodb I don't think we will need the upload stacktrace subtest and cppsuite mv files subtest.  I believe the cppsuite test allows you to set a path to for the test directory to be run in. I also think it is okay and I am not fussy if the test's do get overridden as long as they pass. In the case that they fail, the cppsuite tests should stop with the WT_TEST failure.  Let me know if I misunderstood the interpretation of the functions